pages/common/unzip.md

```diff
@@ -1,24 +1,25 @@
 # unzip
 
-> Extract compressed files in a ZIP archive.
+> Extract compressed files/directories from zip archives.
+> See also: `zip`.
 > More information: <https://manned.org/unzip>.
 
-- Extract zip file(s) (for multiple files, separate file paths by spaces):
+- Extract files/directories from specific archives:
 
-`unzip {{file(s)}}`
+`unzip {{path/to/archive1 path/to/archive2 ...}}`
 
-- Extract zip files(s) to given path:
+- Extract files/directories from archives to a specific path:
 
-`unzip {{compressed_file(s)}} -d {{path/to/put/extracted_file(s)}}`
+`unzip {{path/to/archive1 path/to/archive2 ...}} -d {{path/to/output}}`
 
-- List the contents of a zip file without extracting:
+- Extract files/directories from archives to stdout alongside extracted file/directory names:
```

Comment:
This sounds a little like without this option, the command doesn't print the file names to stdout. How about
```suggestion
- Extract files/directories from archives to `stdout`:
```